src/pkg/cli/waitForCdTaskExit.go

```diff
@@ -20,6 +22,12 @@ func WaitForCdTaskExit(ctx context.Context, provider Provider) error {
 				if errors.Is(err, io.EOF) {
 					// EOF indicates that the task has completed successfully
 					return nil
```

Comment:
Wonder if we should switch the `return nil` and `return err` branches, to be more Go-like